spec/factories.rb

```diff
@@ -56,4 +60,8 @@ class Comment
   trait :with_address do
     address { "address" }
   end
+
+  factory :task do
+    FactoryBot::VERSION >= "6.0.0" ? traits_for_enum(:status, {queued: 0, started: 1, finished: 2}) : trait(:queued) { status { 0 } } && trait(:started) { status { 1 } } && trait(:finished) { status { 2 } }
```

Comment:
@chubchenko this might be easier to read if it's broken out into a simple if/else condition.